services/authentication/src/views/login.ejs

```diff
@@ -0,0 +1,150 @@
+<!DOCTYPE html>
```

Comment:
It looks like @jaredcwhite may be already moving down this road in parallel. but I think we want the UI to live with the apps not with the service, so long as that doesn't make life exceedingly difficult. I expect both the branding and potentially the UX more generally to be pretty different between public and partners, so I'm not sure that we save much by combining any forms.

Reply:
I agree in theory. However, for security reasons, OIDC interactions for authenticating need to happen from the auth server.

That being said, this is just a sample implementation of a login form from the `oidc-provider` library. I'd definitely rather use our components and (obviously) branding, so I'm totally happy to swap it out. However, to use those components I would have had to set up a React app in this service, so I wanted to make sure that we actually wanted to start out with a email/pw form (vs. e.g. Google SSO) before I put in the work to do that.